internal/api/oidc/op.go

```diff
@@ -120,7 +126,15 @@ func NewServer(
 
 	server := &Server{
 		LegacyServer:        op.NewLegacyServer(provider, endpoints(config.CustomEndpoints)),
+		features:            config.Features,
+		repo:                repo,
+		query:               query,
+		command:             command,
+		keySet:              newKeySet(context.TODO(), time.Hour, query.GetActivePublicKeyByID),
+		fallbackLogger:      fallbackLogger,
+		hashAlg:             crypto.NewBCrypt(10), // as we are only verifying in oidc, the cost is already part of the hash string and the config here is irrelevant.
```

Comment:
If it's irrelevant, why do we still configure it here?

Reply:
The meaning of the comment is exactly that I'm not using configuration, as in `defaults.yaml`, because that is irrelevant. But I still need to call the constructor, which always needs the cost parameter. There is no "Verify only" mode in the legacy bcrypt implementation.

I just gave it a value of 10, but any integer >4 is valid.